product_docs/docs/edbcloud/beta/using_cluster/connecting_your_cluster/02_connect_on_premises.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Connecting from Networks Outside Azure
+---
+
+
+If your applications are deployed on your on-premises environments, or on other Cloud providers instead Azure, you can set up a secure tunnel between your on premises network and the Azure virtual network. 
+
+There are a few options you can use.
+
+## Connecting Using Azure VPN Gateway (Point-to-Site)
+
+Azure VPN gateway provides you an encrypted tunnel to communicate
+with your IT infrastructures on a Azure Private Network over complex
+network environments like the internet.
+
+Point-to-Site VPN connections are useful when you want to connect to
+your EDB Cloud cluster from a remote location, such as when you do
+administrative operations from home. You can also use P2S instead of a
+Site-to-Site VPN when you have only a few clients that need to connect
+to your EDB Cloud service. Point-to-Site connections do not require a VPN
+device or a public-facing IP address. P2S creates the VPN connection
+over either SSTP (Secure Socket Tunneling Protocol) or IKEv2 protocol.
+
+The Azure resources that are created when you create a virtual network
+gateway include:
+
+-   A VM that is deployed in your EDB Cloud cluster virtual network. It works as a VPN application to handle all traffic.
+
+-   A dedicated subnet called `GatewaySubnet` to accommodate the VPN VM.
+
+## Point-to-Site Example
+
+These are the steps to connect using the Azure VPN Gateway Point-to-Site option. 
+
+### Step 1: Create Virtual Network VPN Gateway
+
+ In the **Virtual network gateway** page, create the first VPN gateway for the EDB Cloud Cluster resource virtual network `vnet-japaneast`. Name the VPN gateway "vpng-edbcloud."
+
+ ![](images/image8.png)
+
+!!! Note
+    The VPN gateway automatically creates a dedicated subnet to accommodate its gateway VMs. Ensure that your EDB Cloud virtual network address space has sufficient IP range for the subnet to prevent errors in the virtual network.
+
+
+### Step 2: Create self-signed root CA for your VPN gateway authentication
+
+Certificates are used by Azure to authenticate clients connecting to a
+VNet over a Point-to-Site VPN connection. See
+[Generate a Self-Signed Root
+Certificate](https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-howto-point-to-site-resource-manager-portal#getcer).
+This example uses a self-signed certificate, but EDB recommends you use your Enterprise certificate in
```

Comment:
> you use your Enterprise certificate

@zozowell: Is this something that Azure customers should understand? I'm not sure I follow.